FlipperZero/FlipperZero.xcodeproj/project.pbxproj

```diff
@@ -376,6 +378,45 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		497F0FAA25269B8A000B2A86 /* Check code with SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Check code with SwiftLint";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Lint with systemwide installation if present\nif which swiftlint >/dev/null; then\n  swiftlint\n  exit 0\nfi\n# Lint with local swiftlint copy if present\nif [ -f ${PROJECT_DIR}/swiftlint ]; then\n  ${PROJECT_DIR}/swiftlint\nelse\n  # Download a copy of swiftlint to project root and lint afterwards\n  curl -o- -Ls https://github.com/realm/SwiftLint/releases/latest/download/portable_swiftlint.zip | tar xzf - swiftlint \\\n  && chmod +x swiftlint || echo \"warning: Failed to download SwiftLint\"; exit 0\n  ${PROJECT_DIR}/swiftlint \nfi\n";
```

Comment:
Could you please move the script into a separate file, so it'll be easier to maintain?

Reply:
I was waiting for that. :)
If losing access to project's envvars (eg ${PROJECT_DIR}) isn't an issue I'll eliminate the duplication.